src/components/Buttons.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Buttons({onClickBtn, isDisabled, children}) {
+    return (
+        <button type="button" onClick={onClickBtn} disabled={isDisabled}>{children}</button>
+    );
+}
```

Comment:
Dit component is nu een soort van doorgeefluik. Het zorgt niet voor minder code. Je kunt daarentegen direct `button` gebruiken waar nu `Buttons` gebruikt wordt